internal/statemachine/image_definition.go

```diff
@@ -40,8 +43,8 @@ type RootfsType struct {
 	Components   []string     `yaml:"components"    json:"Components,omitempty"`
 	Archive      string       `yaml:"archive"       json:"Archive"                default:"ubuntu"`
 	Flavor       string       `yaml:"flavor"        json:"Flavor"                 default:"ubuntu"`
-	Pocket       string       `yaml:"pocket"        json:"Pocket"                 default:"release"`
 	Mirror       string       `yaml:"mirror"        json:"Mirror"                 default:"http://archive.ubuntu.com/ubuntu/"`
+	Pocket       string       `yaml:"pocket"        json:"Pocket"                 jsonschema:"enum=release,enum=updates,enum=security,enum=proposed" default:"release"`
```

Comment:
A schema question that needs bikeshedding: should the pockets be lowercase or uppercase? In the apt world we always use lowercase (updates), but Launchpad for instance insists on using those uppercase (Updates). I don't have a strong feeling here. What do you think?

Reply:
I'd be happy to accept either upper or lower just so we don't frustrate any users.

Reply:
Pushed an update to allow the first character to be upper or lower case (Updates/updates, release,Release, etc).